conans/test/integration/command/create_test.py

```diff
@@ -409,3 +409,114 @@ def test_create_build_missing():
     c.assert_listed_binary({"pkg/1.0": ("4c0c198b627f9af3e038af4da5e6b3ae205c2435", "Build")})
     c.assert_listed_binary({"dep/1.0": ("9a4eb3c8701508aa9458b1a73d0633783ecc2270", "Missing")})
     assert "ERROR: Missing prebuilt package for 'dep/1.0'" in c.out
+
+
+def test_create_format_json():
+    """
+    Tests the ``conan create . -f json`` result
+
+    The result should be something like:
+
+    {
+        'graph': {
+            'conanfile': {...},  # Consumer
+            'hello/0.1#18d5440ae45afc4c36139a160ac071c7': {...},
```

Comment:
Is this recursive or something? Is the graph represented somehow?

Reply:
Yes, the graph is represented as it is. I got back the original serialization so it's using the whole information from `DepsGraph.serialize()` without pruning anything.